feat: creating kotlin data analysis get started tutorial (#4108)
JetBrains/kotlin-web-site: docs/topics/data-analysis/kotlin-notebook-share.md

[//]: # (title: Share your Kotlin Notebook)

<microformat>

>    <p>This is the fourth part of the <strong>Getting started with Kotlin Notebook</strong> tutorial. Before proceeding, make sure you've completed the previous step.</p>

Reviewer: ```suggestion
   <p>This is the fourth part of the <strong>Getting started with Kotlin Notebook</strong> tutorial. Before proceeding, make sure you've completed the previous steps.</p>
```

   <p><img src="icon-1-done.svg" width="20" alt="First step"/> <a href="kotlin-notebook-set-up-env.md">Set up an environment</a><br/>
      <img src="icon-2-done.svg" width="20" alt="Second step"/> <a href="kotlin-notebook-create.md">Create a Kotlin Notebook</a><br/>
      <img src="icon-3-done.svg" width="20" alt="Third step"/> <a href="kotlin-notebook-add-dependencies.md">Add dependencies to a Kotlin Notebook</a><br/>      
      <img src="icon-4.svg" width="20" alt="Fourth step"/> <strong>Share a Kotlin Notebook</strong><br/>
  </p>
</microformat>

Now that you have created your first Kotlin Notebook, you are ready to share your work with others! 

## Share a Kotlin Notebook

> To share a Kotlin Notebook, you can simply upload it to any notebook web viewer, since Kotlin notebooks adhere to the 

Reviewer: ```suggestion
To share a Kotlin Notebook, just upload it to any notebook web viewer because Kotlin notebooks follow the 
```

Reviewer: Simplified a bit.

Reviewer: Simplified a bit.

universal Jupyter format.

One effective platform is [JetBrains Datalore](https://datalore.jetbrains.com). 

> This platform not only facilitates the sharing of Kotlin notebooks but also enhances their usability.

Reviewer: In the "improvements vs enhancement" battle, improvements always win!
See the guidelines for details.

Datalore allows you to execute and edit notebooks, and incorporates advanced features like the creation of interactive 

> reports and the option to schedule notebook runs. To see it in action, see [Kotlin Datalore example using DataFrame](https://datalore.jetbrains.com/report/static/KQKedA4jDrKu63O53gEN0z/B5YeMMONSAR78FgKQ9yJyW).

Reviewer: Does it work?

> reports and the option to schedule notebook runs. To see it in action, see [Kotlin Datalore example using DataFrame](https://datalore.jetbrains.com/report/static/KQKedA4jDrKu63O53gEN0z/B5YeMMONSAR78FgKQ9yJyW).

Author: it does - I like it, a bit more concise and flows just as well if not better

![Datalore Notebook example](datalore-example.png){width=700}

> GitHub is another great option. It natively renders Kotlin notebooks, allowing for straightforward sharing and collaboration.

Reviewer: Also, let's also remove "great" and "effective" words from this two paras.

> GitHub is another great option. It natively renders Kotlin notebooks, allowing for straightforward sharing and collaboration.

Author: sounds good to me, I'm including:
"We recommend the following platforms to share Kotlin notebooks:" and a putting the GitHub and Datalore as bullet points
